autonat/autonat-v2.md

```diff
@@ -167,7 +180,15 @@ message DialRequest {
 }
 
 message DialAttempt {
-    fixed64 nonce = 1;
+  fixed64 nonce = 1;
+}
+
+message DialDataRequest {
+  uint64 numBytes = 1;
+}
+
+message DialDataResponse {
+  bool accepted = 1;
```

Comment:
Why not put the bytes into this protobuf? Then we wouldn't need to break the framing to send these bytes.

Reply:
That would require memory of the order of 100kb for the protobufs on both ends, right? 
The current version can be implemented using a smaller buffer.

I don't have good intuition here, do you think this is fine?

Reply:
@marten-seemann friendly ping

Reply:
The server is going to discard these bytes right away anyways. No point to allocate the space to put the bytes in the message. I agree with the initial design. (Sorry for the drive-by comment, I'll defer to whatever you all decide).